app/controllers/currentgame_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CurrentgameController < ApplicationController
```

Comment:
Since the "actions" won't overlap between `CurrentgameController` and `NewgameController`, would it be possible to combine these into a single `GamesController`?  Then `index`, `show`, `new`, and `create` can all be in the same file.

That'll also have the added benefit of allowing us to simply add `resources :games` in `routes.rb` rather than anything more complicated.